src/scripting/lua_widget_methods.cpp

```diff
@@ -359,55 +400,59 @@ static int intf_set_dialog_focus(lua_State* L)
 /**
  * Sets a widget's state to active or inactive
```

Comment:
This is one of at least three that look like they were just copied but don't apply to the function at hand.